include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -161,6 +161,57 @@ def InstanceOp : HardwareDeclOp<"instance", [
   }];
 }
 
+def InstanceChoiceOp : HardwareDeclOp<"instance_choice", [
+    DeclareOpInterfaceMethods<SymbolUserOpInterface>
+  ]> {
```

Comment:
Why doesn't this implement the same traits/interfaces as the existing `InstanceOp`, e.g., `FInstanceLike` and `InstanceGraphInstanceOpInterface`?

Reply:
It will in an upcoming PR. Some more changes are required for that and it's easier to do if the op is present & tested.

Reply:
Sounds good. I'm fine with that. Thanks for clarifying that the PR isn't overlooking anything.